inbox/react/migration-guide.mdx

```diff
@@ -0,0 +1,737 @@
+  {({ unseenCount }) => <NotificationBell unseenCount={unseenCount} />}
+</PopoverNotificationCenter>
+```
+
+- New Implementation with @novu/react and Radix UI as an example
+
+```tsx
+import React from "react";
+import * as RadixPopover from "@radix-ui/react-popover";
+import { Inbox, Bell, Notifications } from "@novu/react";
+
+function Novu() {
+  return (
+    <Inbox
+      applicationIdentifier="YOUR_APP_ID"
+      subscriberId="YOUR_SUBSCRIBER_ID"
+    >
+      <RadixPopover.Root>
+        <RadixPopover.Trigger asChild>
+          <Bell />
+        </RadixPopover.Trigger>
+        <RadixPopover.Portal>
+          <RadixPopover.Content side="bottom" align="end" sideOffset={10}>
+            <Notifications />
+            <RadixPopover.Arrow />
+          </RadixPopover.Content>
+        </RadixPopover.Portal>
+      </RadixPopover.Root>
+    </Inbox>
+  );
+}
+
+export default Novu;
+```
+
+## Custom Notification Item
+
+Customize the appearance and structure of individual notification items using the `renderNotification` prop.
+
+- Old Implementation
+
+```tsx<PopoverNotificationCenter
+  colorScheme={colorScheme}
+  onNotificationClick={handlerOnNotificationClick}
+  onActionClick={handlerOnActionClick}
+  listItem={(
+    notification,
+    handleActionButtonClick,
+    handleNotificationClick
+  ) => {
+    return (
+      <a
+        href="/"
+        onClick={(e) => {
+          e.preventDefault();
+          handleNotificationClick();
+        }}
+      >
+        {notification.content}
+      </a>
+    );
+  }}
+>
+  {({ unseenCount }) => {
+    return <NotificationBell unseenCount={unseenCount} />;
+  }}
+</PopoverNotificationCenter>
+```
+
+- New Implementation with @novu/react
+
+```tsx
+import { Inbox } from "@novu/react";
+
+function Novu() {
+  const renderCustomNotificationItem = (notification) => (
+    <div
+      className={`notification-item ${notification.isRead ? "read" : "unread"}`}
+      onClick={() => notification.read()}
+    >
+      <img
+        src={notification.avatar}
+        alt="Avatar"
+        className="notification-avatar"
+      />
+      <div className="notification-content">
+        <h4 className="notification-subject">{notification.subject}</h4>
+        <p className="notification-body">{notification.body}</p>
+      </div>
+    </div>
+  );
+
+  return (
+    <Inbox
+      applicationIdentifier="YOUR_APP_ID"
+      subscriberId="YOUR_SUBSCRIBER_ID"
+      renderNotification={renderCustomNotificationItem}
+    />
+  );
+}
+
+export default Novu;
+```
+
+## Styling with appearance Prop
+
+Customize the overall look and feel of the notification components using the appearance prop, which supports both CSS objects and class names (including Tailwind CSS classes).
+
+- Old Implementation
+
+```tsx
+<div className="bell">
+  <NovuProvider
+    subscriberId={process.env.REACT_APP_SUB_ID}
+    applicationIdentifier={process.env.REACT_APP_APP_ID}
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Let's use everywhere the same example for these fields:
```
applicationIdentifier="YOUR_APP_ID"
subscriberId="YOUR_SUBSCRIBER_ID"
```